src/GovUk.Education.ExploreEducationStatistics.Data.Api/Models/Query/ObservationQueryContext.cs

```diff
@@ -8,6 +8,7 @@ public class ObservationQueryContext
         public long SubjectId { get; set; }
         public TimePeriodQuery TimePeriod { get; set; }
         public IEnumerable<long> Filters { get; set; }
+        public long? BoundaryLevel { get; set; }
```

Comment:
Is it okay for this to be null?

Reply:
Yes it is. If it's not populated we work out the latest set of boundary data. It also means it's backwards compatible with the existing chart configuration.